server.js

```diff
@@ -0,0 +1,14 @@
+const express = require ('express')
+const app = express();
+const port = 3003;
```

Comment:
It's a good practice to put your port number in an environment variable. This allows for more flexibility, especially when deploying your application to different environments. [medium]